### ISSUE TITLE
Any plan for training code?

### ISSUE THREAD

**Issue by hdjsjyl** [IDEA-Research/MotionLLM#1]
Hi Author,
    Thanks for sharing your excellent work. Is there any plan for releasing training code?

**Comment by LinghaoChan** [comment]
> Hi Author, Thanks for sharing your excellent work. Is there any plan for releasing training code?

@hdjsjyl Thanks for your interest. 

Aha, we do wanna plan to release the training codes. However, the training cost is heavy (~8 A100-80GB). But I think this might not be the most challenging part for you. The dataset preparation is much heavier, due to the AMASS license and licenses of some Motion-X subsets. I definitely know releasing the training codes will be useful to you. But this might take you 1+ month to prepare and it is also a bit difficult for us to prepare the detailed document. We now plan to release some examples to tell the community how to tune MotionLLM. Do you think this is acceptable? If there is a strong need on the training codes, we will try our best to do so.

**Comment by hdjsjyl** [comment]
hi @LinghaoChan ，
    Thanks for your detailed reply. 
    1. The GPU resource is not a problem;
    2. Example of tunning MotionLLM is a good start. 
    Really appreciate your work.

**Comment by LinghaoChan** [comment]
> hi @LinghaoChan ， Thanks for your detailed reply. 1. The GPU resource is not a problem; 2. Example of tunning MotionLLM is a good start. Really appreciate your work.

Great. I add this to my plan.

**Comment by vvirgooo2** [comment]
Hi author, I want to know when examples of tunning MotionLLM can be released.  Can you provide a rough time? Thanks！

**Comment by LinghaoChan** [comment]
I think this might be 1-2 months.

**Comment by 41xu** [comment]
Hey, when do you plan to release your training code? @LinghaoChan

**Comment by LinghaoChan** [comment]
> Hey, when do you plan to release your training code? @LinghaoChan 

tomorrow we will release a new project. Your issue will be our next milestone.

**Comment by 41xu** [comment]
> > Hey, when do you plan to release your training code? @LinghaoChan
> 
> tomorrow we will release a new project. Your issue will be our next milestone.

OK, is this [MotionCLR](https://github.com/IDEA-Research/MotionCLR)? so what about the training code of this MotionLLM🌚

**Comment by LinghaoChan** [comment]
> > > Hey, when do you plan to release your training code? @LinghaoChan
> 
> > 
> 
> > tomorrow we will release a new project. Your issue will be our next milestone.
> 
> 
> 
> OK, is this [MotionCLR](https://github.com/IDEA-Research/MotionCLR)? so what about the training code of this MotionLLM🌚

Exactly. We set the MotionLLM data release in our first priority. The training codes will come after this.
